spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -108,6 +111,8 @@
 
     private static final String GITHUB_API_EMAIL = GITHUB_API_USER + "/emails";
 
```

Comment:
I would remove the blank line between the variables.

Reply:
Done